app/tasks/last_applicant_answers_fetcher.rb

```diff
@@ -4,7 +4,7 @@ class LastApplicantAnswersFetcher < ApplicationTask
   def initialize(current_user)
     @current_user = current_user
     @user_detail = current_user.user_detail
-    @job_applications = current_user.job_applications.where(status: "completed")
+    @job_applications = current_user.job_applications # .where(status: "completed")
```

Comment:
Why has this changed? What is its impact?

Reply:
Temporary solution to be able to test pre-filler and see the gap between array version to hash version.
Hash broke many things and most of the application_forms are not going to the state completed. So i remove it to be able to have more information to be able to debug.